etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/utils/security/SecureHash.java

```diff
@@ -0,0 +1,7 @@
+package gov.hhs.cdc.trustedintermediary.etor.utils.security;
+
+import java.security.NoSuchAlgorithmException;
+
+public interface SecureHash {
```

Comment:
I agree with Peter. In general, I think it's better to avoid adding code preemptively as there's always a cost to adding more code (adds complexity, which increases potential for bugs, mental overhead, etc); and we can always refactor to add an interface if needed in the future

Reply:
That makes sense. I removed it